LoopCaregiverKit/Sources/LoopCaregiverKitUI/LatestGlucoseRectangularView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LatestGlucoseRectangularView.swift
+//  LoopCaregiverWidgetExtension
+//
+//  Created by Auggie Fisher on 1/24/24.
+//
+
+import HealthKit
+import LoopCaregiverKit
+import LoopKit
+import SwiftUI
+
+public struct LatestGlucoseRectangularView: View {
+
+    
+    public let viewModel: WidgetViewModel
+    @Environment(\.colorScheme) var colorScheme
+    
+    public init(viewModel: WidgetViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        
+        HStack (spacing: 10) {
+            VStack {
+                //BG number
+                Text(viewModel.currentGlucoseNumberText)
+                    .foregroundStyle(egvColor)
+                    .strikethrough(viewModel.isGlucoseStale)
+                    .font(.system(size: 65))
+            }
+            VStack (spacing: 0) {
+                HStack {
+                    //Trend arrow
+                    if let currentTrendImageName = viewModel.currentTrendImageName {
+                        Image(systemName: currentTrendImageName)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(maxWidth: 12)
+                            .offset(.init(width: 0.0, height: 1.5))
+                    }
+                    //BG delta
+                    Text(viewModel.lastGlucoseChangeFormatted!)
+                        .strikethrough(viewModel.isGlucoseStale)
+                        .font(.system(size: 20.0))
+                }
+                //Minutes since update
```

Comment:
I feel the ~ symbol takes up the already limited real-estate on the watch and may not be worth it -- especially if it is within 1 minute. Tell me more though if there is more to this.